mesheryctl/internal/cli/root/config/config.go

```diff
@@ -55,8 +55,8 @@ func GetMesheryCtl(v *viper.Viper) (*MesheryCtlConfig, error) {
 	return c, err
 }
 
-// SetMesheryCtl sets the mesheryctl configuration object
-func SetContext(v *viper.Viper, context *Context, name string) error {
+// UpdateContextToConfig write the given context in meshconfig
+func UpdateContextToConfig(v *viper.Viper, context *Context, name string) error {
```

Comment:
Minor: UpdateContextInConfig or UpdateConfigWithContext sounds more readable to me